ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/brew/tasks/brew.yml

```diff
@@ -0,0 +1,85 @@
+---
+
+- name: Check if Homebrew is already installed
+  stat:
+    path: /usr/local/bin/brew
+  register: b
+  when: ansible_architecture == "x86_64"
+  tags: brew
+
+- name: Check if Homebrew is already installed
+  stat:
+    path: /opt/homebrew/bin/brew
+  register: b
+  when: ansible_architecture == "arm64"
+  tags: brew
+
+- name: Set brew_is_installed
+  set_fact: brew_is_installed="{{ b.stat.exists }}"
+  when: b.stat.exists is defined
+  tags: brew
+
+- name: Install home brew
+  when: not brew_is_installed
+  tags: brew
+  block:
+    - name: Add user j9admin to sudoers
```

Comment:
This seems like a OpenJ9 specific Admin user, I'm not sure this is warranted (even if it is cleanup later).